src/main/app/App.js

```diff
@@ -1,21 +1,23 @@
-import React, { Component } from 'react';
-import logo from '../../assets/logo.svg';
+import React from 'react';
+import { App } from 'grommet';
+import '../../../node_modules/grommet-css';
```

Comment:
Just `import 'grommet-css';`

Reply:
nope, it's necessary to import  /node_modules/grommet-css
https://www.npmjs.com/package/grommet-css

Reply:
are you sure? I tried using just 'grommet-css', and it seems like everything is ok

Reply:
try npm install anew and check, refresh page any times
could be all seems like everything is ok cause you tried it after installing 'grommet'